clang/lib/CodeGen/CGBuiltin.cpp

```diff
@@ -16542,12 +16542,64 @@ Value *CodeGenFunction::EmitPPCBuiltinExpr(unsigned BuiltinID,
 
   Intrinsic::ID ID = Intrinsic::not_intrinsic;
 
+  // The lambda function converts builtin_cpu_is function into directly
+  // returning false or true, or it gets and checks the information from the
+  // kernel variable _system_configuration for AIX OS.
```

Comment:
```suggestion
  // This lambda function converts __builtin_cpu_is() into directly
  // returning true or false, or it gets and checks the information from the
  // kernel variable _system_configuration from the AIX OS.
```
I'm not sure if this wording sounds better or more accurate, so we can change this but basically I think we can update this comment a bit more so that it makes more sense.